clients/js/sui.ts

```diff
@@ -58,13 +62,18 @@ export async function callEntryFunc(
     args: Array<any>,
 ) {
     let signer = loadSigner(network, rpc);
+    console.log("network: ", network)
+    console.log("rpc: ", rpc)
+    console.log("package object id: ", packageObjectId)
+    console.log("module: ", module)
     const moveCallTxn = await signer.executeMoveCall({
         packageObjectId: packageObjectId,
         module: module,
         function: func,
         typeArguments: type_args,
         arguments: args,
-        gasBudget: 20000,
-      });
-      console.log('moveCallTxn', moveCallTxn);
+        gasBudget: 50000,
+    });
+    console.log('moveCallTxn: ', moveCallTxn);
+    console.log("effects: ", util.inspect(moveCallTxn["EffectsCert"]["effects"], {showHidden: false, depth: null, colors: true}))
```

Comment:
passing `depth: null` means this will recurse indefinitely if there is ever a circular reference in the object

i'd prefer to get rid of the util import and JSON stringify the obj isntead
```suggestion
    console.log("effects: ", JSON.stringify(moveCallTxn["EffectsCert"]["effects"], null, 2))
```